spec.html

```diff
@@ -34651,7 +34651,7 @@ <h1>_TypedArray_.prototype</h1>
 
     <!-- es6num="22.2.6" -->
     <emu-clause id="sec-properties-of-typedarray-prototype-objects">
-      <h1>Properties of _TypedArray_ Prototype Objects</h1>
+      <h1>Properties of the _TypedArray_ Prototype Objects</h1>
```

Comment:
also here

Reply:
There's already `Properties of the _NativeError_ Prototype Objects`, plus the ones mentioned above.